lib/modules/subredditManager.js

```diff
@@ -821,217 +828,48 @@ function redrawSubredditBar() {
 		}
 	}
 	const originalFrontpage = originalShortcuts.find(({ pathname }) => pathname === '/');
-	const originalPopular = originalShortcuts.find(({ pathname }) => pathname === '/r/popular/');
+	const originalAll = originalShortcuts.find(({ pathname }) => pathname === '/r/all');
+	const originalPopular = originalShortcuts.find(({ pathname }) => pathname === '/r/popular');
 	const originalProfilePosts = originalShortcuts.find(({ pathname }) => pathname === '/r/profileposts/');
+	const originalOC = originalShortcuts.find(({ pathname }) => pathname === '/original');
 
-	const headerContents = document.querySelector('#sr-header-area');
-	if (headerContents) {
-		// Clear out the existing stuff in the top bar first, we'll replace it with our own stuff.
-		$(headerContents).html('');
+	$("#hamburgers").show();
```

Comment:
I appreciate the effort you put into this! but it'll make for kinda weird UX and might be fragile. I'm hoping to get JSAPI integration for the hamburger menu set up soon that will make this much simpler.

Reply:
If you have a free moment, it would be nice to make a separate PR with just the `originalOC` changes, that could get merged sooner!

Reply:
OK I will split the `originalOC` code and pull it in